kdwsdl2cpp/src/main.cpp

```diff
@@ -67,6 +68,12 @@ static void showHelp(const char *appName)
             "                            use of the import-path option\n"
             "  -help-on-missing          When groups or basic types could not be found, display\n"
             "                            available types (helps with wrong namespaces)\n"
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
```

Comment:
I don't like it that the option is dependent on de Qt version. However I don't know how to do it better ;-)

Reply:
Before Qt 5.4 I used QCA for this purpose, but I do not think that we should go down that way.